src/nginxconfig/generators/conf/drupal.conf.js

```diff
@@ -32,6 +33,11 @@ export default global => {
         deny: 'all',
     };
 
+    config['# Drupal: allow image styles to be handled by the CMS'] = '';
+    config['location ~ ^/sites/[^/]+/files/styles/'] = {
+        try_files: '$uri /index.php?q=$uri&$args;',
```

Comment:
```suggestion
        try_files: '$uri /index.php?q=$uri&$args',
```
Don't need the semi at the end, the conf generator will add that in